ecs-init/cache/cache.go

```diff
@@ -61,6 +67,19 @@ func (d *Downloader) fileNotEmpty(filename string) bool {
 	return fileinfo.Size() > 0
 }
 
+// getRegion finds region from metadata and caches for the life of downloader
+func (d *Downloader) getRegion() string {
+	if d.region == "" {
+		region, ok := d.metadata.Region()
```

Comment:
This should be `region, err := `.. `ok` is used when a `bool` is being returned.